docs/faq.mdx

```diff
@@ -29,11 +29,11 @@ Mock Service Worker provides a Node-compatible API to allow you to reuse the sam
 
 ## Can I use it with React Native?
 
-Yes, but only for NodeJS integration.
+Yes, but because MSW relies on running a Node process, it can only be used for local development in the simulator or running unit/integration tests.
 
-Since React Native does not execute in a browser environment, you cannot run a Service Worker alongside your application.
+Since React Native does not execute in a browser environment, you will need to add polyfills to patch some of the browser API, by adding `react-native-url-polyfill` and `import { setupServer } from 'msw/native'`.
+For running unit/integration tests, use `import { setupServer } from 'msw/node'`.
```

Comment:
This is an interesting distinction, I haven't thought of this. Thanks for this clarification 👍